### PR TITLE
Feat : Login and signup pages

### PR DESCRIPTION
**Pull request by yogesh-**
Login and Signup pages completed.

Preview - Login page
https://deploy-preview-5--sneaker-don.netlify.app/html/login.html

Preview - Singup Page
https://deploy-preview-5--sneaker-don.netlify.app/html/signup.html

image

image

### CODE REVIEW

File: css/login-signup.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  border-radius: 4px;
+  color: var(--text-color);
+}
+
+.login-form .form > p {
+  margin-bottom: 3rem;
+}
+
+/* .first-input {
+  margin: 0 auto;
```
Comment (chanchal-panpaliya) [inline css/login-signup.css]: remove unwanted code .

**Review by akash-1318 — COMMENTED** [review]
Looks good just one suggestion you are not using code formatter and also i hope you will remove this commented code else everything is fine.

File: html/login.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    <div class="login-container">
+        <nav class="navigaton flex-row">
+
+            <div class="logo">
+
+                <p class="h1"><a href="/index.html">SNEAKERS</a></p>
+
+            </div>
+
+            <!-- <div class="search-bar flex-row">
```
Comment (chanchal-panpaliya) [inline html/login.html:25]: remove unwanted code

File: html/signup.html [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+        <nav class="navigaton flex-row">
+
+            <div class="logo">
+
+                <p class="h1"><a href="/index.html">SNEAKERS</p></a>
+                </a>
+            </div>
+
+            <!-- <div class="search-bar flex-row">
+            <input type="text" placeholder="Search">
```
Comment (chanchal-panpaliya) [inline html/signup.html:28]: remove unwanted code.

File: html/signup.html [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+                </a>
+            </div>
+
+            <p class="h5">&#169; 2022 SNEAKERS Pvt Ltd</p>
+        </div>
+    </div>
+
+    <script>
+        function hideUnhide() {
+            var mm = document.getElementById("password");
```
Comment (chanchal-panpaliya) [inline html/signup.html]: Please give it a meaningful name.

please avoid "var" (a global variable) and use "let".

**Review by chanchal-panpaliya — COMMENTED** [review]
code looks good .

File: html/signup.html (hunk 4 above)
Comment (sanjitsarkar) [reply html/signup.html]: Or you can use const if the value of that variable is not changing

**Review by sanjitsarkar — APPROVED** [review]

File: html/signup.html (hunk 4 above)
Comment (yogesh-) [reply html/signup.html]: okay thanks. changed it to const.

**Pull request merged**